From c64571759c10a00b12e7eff9c9d99e02834c0011 Mon Sep 17 00:00:00 2001
From: RezzedUp <RezzedUp@users.noreply.github.com>
Date: Fri, 26 Apr 2024 14:43:14 -0500
Subject: [PATCH] Remove spigot's java version check


diff --git a/src/main/java/org/bukkit/craftbukkit/Main.java b/src/main/java/org/bukkit/craftbukkit/Main.java
index 56ac679b2..c776f13e1 100644
--- a/src/main/java/org/bukkit/craftbukkit/Main.java
+++ b/src/main/java/org/bukkit/craftbukkit/Main.java
@@ -171,11 +171,13 @@ public class Main {
                 return;
             }
 
-            float javaVersion = Float.parseFloat(System.getProperty("java.class.version"));
-            if (javaVersion > 57.0) {
-                System.err.println("Unsupported Java detected (" + javaVersion + "). Only up to Java 13 is supported.");
-                return;
-            }
+            // Cubed start - Remove spigot's java version checker...
+            // float javaVersion = Float.parseFloat(System.getProperty("java.class.version"));
+            // if (javaVersion > 57.0) {
+            //     System.err.println("Unsupported Java detected (" + javaVersion + "). Only up to Java 13 is supported.");
+            //     return;
+            // }
+            // Cubed end
 
             try {
                 // Paper start - Handled by TerminalConsoleAppender
-- 
2.39.3 (Apple Git-146)

